traffic_ops/app/lib/Schema/Result/DeliveryServiceInfoForDomainList.pm

```diff
@@ -86,6 +87,7 @@ __PACKAGE__->add_columns(
 	"multi_site_origin",           { data_type => "integer", is_nullable => 0 },
 	"ds_id",                       { data_type => "integer", is_nullable => 0 },
 	"dscp",                        { data_type => "integer", is_nullable => 0 },
+	"routing_name",                { data_type => "varchar", is_nullable => 0, size => 45 },
```

Comment:
is it 45 or 48? i saw this check in the api

+			routingName          => [ \&is_valid_routing_name, is_long_at_most( 48, 'too long' ) ],

Reply:
Yeah, it's 48. I put 45 here to make it consistent with `xml_id`, which also has a check for 48 in the API but is 45 here. I'll set it to 48 for `routing_name` to match the API.